pyomo/contrib/solver/plugins.py

```diff
@@ -39,7 +39,7 @@ def load():
         doc='Direct interface to Gurobi accommodating general MINLP',
     )(GurobiDirectMINLP)
     SolverFactory.register(
-        name="highs", legacy_name="highs", doc="Persistent interface to HiGHS"
+        name="highs", legacy_name="highs_v2", doc="Persistent interface to HiGHS"
```

Comment:
Why is this being renamed?  There wasn't a `highs`, so it doesn't make a lot of sense to change this to `v2`?